test/commands/site_updates/process_new_exercise_update_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class ConceptExercise::CreateTest < ActiveSupport::TestCase
```

Comment:
```suggestion
class SiteUpdates::ProcessNewExerciseUpdateTest < ActiveSupport::TestCase
```

Reply:
Thanks. I reckon loads of flakey tests happen because of this bug.